web/wp-content/themes/dctx/archive.php

```diff
@@ -1,18 +1,20 @@
 <?php get_header(); ?>
 
 <div id="main">
+	<?php the_archive_title('<h1 class="page-title">', '</h1>'); ?>
+
 	<div class="container clearfix">
 		
```

Comment:
:warning: **Warning**: Found precision alignment of 1 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).